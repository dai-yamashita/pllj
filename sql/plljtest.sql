\set ECHO none
CREATE EXTENSION pllj;
\set ECHO all

do $$
print("Hello LuaJIT FFI")
$$ language pllj;

create table pg_temp.test(txt text);

do $$
local spi = require("pllj.spi")
spi.execute("insert into pg_temp.test(txt) values('qwerty')")
$$ language pllj;

select * from pg_temp.test;
