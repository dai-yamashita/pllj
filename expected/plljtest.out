\set ECHO none
do $$
print("Hello LuaJIT FFI")
$$ language pllj;
INFO:  Hello LuaJIT FFI
create table pg_temp.test(txt text);
do $$
local spi = require("pllj.spi")
spi.execute("insert into pg_temp.test(txt) values('qwerty')")
$$ language pllj;
select * from pg_temp.test;
  txt   
--------
 qwerty
(1 row)

